apps/webapp/web/static/js/docs/DocPage.js

```diff
@@ -1,11 +1,18 @@
 // file: docs/DocPage.js
 import React from 'react';
+import Relay from 'react-relay';
```

Comment:
You don't need to import Relay because you're not using Relay for this component.